.github/workflows/release_engine.yml

```diff
@@ -72,11 +72,16 @@ jobs:
           echo Release Tag: ${{ steps.draft_release.outputs.tag_name }}
           echo Release HTML URL: ${{ steps.draft_release.outputs.html_url }}
       
-      - name: Diagnostics for Beta 
+      # For beta releases we need to craete the beta release first so that 
+      # jobs and steps that follow can upload artifacts.  
+      - name: Create Release for Beta 
```

Comment:
We should create it with the same description as the drafter generates, ideally? Also, we should make sure that creating a beta release doesn't cause the actual release to draft based only on the changes after the beta.

Reply:
I do have to test that. But, the draft is already created (even now for version 0.9.0) so that will at least have the PRs up to the beta release creation. 

I am not sure if adding the same description will cause more issues?